### ISSUE TITLE
[ENDPOINT] Proposed schema change for getListings

### ISSUE THREAD

**Issue by nmrdxt** [etsy/open-api#17]
**Current Endpoint Schema**
I think a way to filter listings by keyword is missing from the current schema. Also, on a lower priority, filtering listings by section ID would be nice to have.

**Proposed Endpoint Schema** 
Adding a query parameter 'keyword' (or any other contextually correct term :)). For the section ID, a section_id parameter could either be an integer or an array of integers for multiple sections.

**Why are you proposing this change?**
Including a way to filter queries would not only reduce the number of calls devs would need to make, but it would also reduce bandwidth and load in the long run due to reduced payload sizes and the need to paginate.

**Comment by benjymouse** [comment]
+1 for the section ID

Have you looked at findAllListingsActive for keyword filter?

**Comment by nmrdxt** [comment]
Yes, I did. I like the level of specificity you can get with findAllListingsActive. I wish there was more control with the getListings endpoint, as I feel like the ability to filter through a shop's listings (including the non-acitve listings) could be very useful to some devs (me included hehe).

**Comment by benjymouse** [comment]
Oh, good point! :)

**Comment by etsyachristensen** [comment]
@nmrdxt We have an endpoint for shop sections: [getListingsByShopSectionId](https://developers.etsy.com/documentation/reference#operation/getListingsByShopSectionId). This should satisfy the section id filter. As for the keyword thing, we will add that to the list of topics to discuss for the next few months of work.
